modules/insights-operator-register-unconnected-cluster.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc
+// * sd_support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc
+
+
+:_content-type: PROCEDURE
+[id="insights-operator-register-unconnected-cluster_{context}"]
+= Registering your unconnected cluster
```

Comment:
s/unconnected/disconnected throughout ??

Reply:
Well spotted. "Disconnected" is correct. I'll update all my instances.